tools/mkmapdata/mkmapdata.sh

```diff
@@ -288,7 +279,7 @@ else
 fi
 
 # Set timestamp for names below 
-CDATE="c"`date +%y%m%d`
+CDATE="c"`date -d "-0 days" +%y%m%d`
```

Comment:
I don't see that we need this change.

Reply:
This was convenient for me when rerunning mkmapdata.sh because I didn't have to recreate all the map_ files from scratch. For example a couple of days ago I set this line to `-28 days` and focused on the files that hadn't been created, yet. I'm happy to revert to its original form, but I found this `days` flag to be a huge time saver.

Reply:
OK, if you want to keep it, add a comment that explains why it's there and how it can help, and how you would use it. Otherwise, it's extra syntax that doesn't actually do anything.